src/sha256t.rs

```diff
@@ -42,8 +42,8 @@ impl<T: Tag> HashTrait for Hash<T> {
     type Engine = sha256::HashEngine;
     type Inner = [u8; 32];
 
-    fn engine() -> sha256::HashEngine {
```

Comment:
Since the hash type is `sha256t`, it's for tagged sha256 hashes. So this can stay as the engine.